styles/Editor.css

```diff
@@ -751,12 +747,12 @@
 
 .ProseMirror .csl-left-margin {
   padding-right: 1ch;
-  display: none;
+  display: table-cell;
 }
 
-.ProseMirror .csl-right-inline::before {
-  content: counter(bib_item) '. ';
-}
+/* .ProseMirror .csl-right-inline {
```

Comment:
Do we have any particular reason not to remove it completely?

Reply:
No reason :) I will remove it.

Reply:
This is removed now.